### PR TITLE
Update(music docs)

### PR DESCRIPTION
**Pull request by USERSATOSHI**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/converter.js
```
Comment (Leref) [inline .gitignore:1]: Remove this.

File: src/guides/@akarui/aoi.music.md [hunk 2]
```diff
@@ -45,6 +50,19 @@ const voice = new AoiVoice(bot, {
         soundcloudLikeTrackLimit: 200,
     },
 });
+// adds a cacher plugin
+voice.addPlugin(PluginName.Cacher, new Cacher("memory" /* or "disk" */));
```
Comment (Leref) [inline src/guides/@akarui/aoi.music.md:54]: Space / break here

**Review by Leref — CHANGES_REQUESTED** [review]
Needed changes.

File: src/guides/@akarui/aoi.music.md (hunk 2 above)
Comment (USERSATOSHI) [reply src/guides/@akarui/aoi.music.md:54]: done

**Comment by Leref** [comment]
Fix the conflicting.

**Comment by USERSATOSHI** [comment]
> Fix the conflicting.

i can't see what lines are conflicting with each other as i dont have write access

File: src/guides/akarui/aoi.music.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
 ---
-title: aoi.music
+
+title: akarui/aoi.music
```
Comment (Leref) [inline src/guides/akarui/aoi.music.md:3]: Keep only topic name.
just `aoi.music`.

File: src/guides/akarui/aoi.music.md [hunk 4]
```diff
@@ -1,7 +1,9 @@
...
 description: How to integrate aoi.music into your Discord Bot with ease.
-id: aoi.music
+id: akarui/aoi.music
```
Comment (Leref) [inline src/guides/akarui/aoi.music.md:5]: keep the name only of the topic

File: src/guides/akarui/aoi.music.md [hunk 5]
```diff
@@ -49,6 +74,7 @@ const voice = new AoiVoice(bot, {
 <summary>Voice#devOptions</summary>
 
 ```typescript
+
```
Comment (Leref) [inline src/guides/akarui/aoi.music.md:77]: Don't escape

**Review by Leref — CHANGES_REQUESTED** [review]
Needed before approving.

File: src/guides/akarui/aoi.music.md [hunk 6]
```diff
@@ -1,7 +1,9 @@
 ---
+
```
Comment (Leref) [inline src/guides/akarui/aoi.music.md:2]: Remove

File: src/guides/akarui/aoi.music.md [hunk 7]
```diff
@@ -1,7 +1,9 @@
...
 title: aoi.music
 description: How to integrate aoi.music into your Discord Bot with ease.
 id: aoi.music
+
```
Comment (Leref) [inline src/guides/akarui/aoi.music.md:6]: Remove

File: src/guides/akarui/aoi.music.md [hunk 8]
```diff
@@ -79,6 +104,7 @@ devOptions ? : {
     youtubePlaylistLimit? : number;
     spotifyPlaylistLimit? : number;
 };
+
```
Comment (Leref) [inline src/guides/akarui/aoi.music.md:107]: Remove

**Review by Leref — CHANGES_REQUESTED** [review]
Last thing to be accepted.

**Pull request merged**